build/update-codecgen.sh

```diff
@@ -1,157 +0,0 @@
-#!/bin/bash
```

Comment:
ugorji codec was removed from the core, AFAIK `mailru/easyjson` is being used now